src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,101 @@
+  | `disable_puma_rack`                       | `instrumentation.puma_rack`               | `instrumentation.puma_rack: disabled`                                               |
+  | `disable_puma_rack_urlmap`                | `instrumentation.puma_rack_urlmap`        | `instrumentation.puma_rack_urlmap: disabled`                                        |
+  | `disable_rack`                            | `instrumentation.rack`                    | `instrumentation.rack: disabled`                                                    |
+  | `disable_rack_urlmap`                     | `instrumentation.rack_urlmap`             | `instrumentation.rack_urlmap: disabled`                                             |
+  | `disable_redis`                           | `instrumentation.redis`                   | `instrumentation.redis: disabled`                                                   |
+  | `disable_redis_instrumentation`           | `instrumentation.redis`                   | `instrumentation.redis: disabled`                                                   |
+  | `disable_resque`                          | `instrumentation.resque`                  | `instrumentation.resque: disabled`                                                  |
+  | `disable_sinatra`                         | `instrumentation.sinatra`                 | `instrumentation.sinatra: disabled`                                                 |
+  | `disable_rake`                            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_rake_instrumentation`            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_typhoeus`                        | `instrumentation.typhoeus`                | `instrumentation.typhoeus: disabled`                                                |
+
+
+## Removed Support for Ruby 2.2 and 2.3 [#ruby_2223]
+
+  Ruby 2.2 and 2.3 are no longer supported by the ruby agent. In order to continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above.
+
+
+## Enabled Thread Tracing by Default [#thread_tracing]
+
+  The configuration option `instrumentation.thread.tracing` is now enabled by default. This will allow the agent to properly monitor code that occurs inside of threads. Work being done inside of threads that previously was not being recorded will now be recorded and associated with the transaction the thread was created in.
+  If you are currently using custom instrumentation to start a new transaction inside of threads, this may be a breaking change, as it will no longer start a new transaction if one already exists.
+
+
+## Removed Deprecated Instrumentation [#removed_gem_instrumentation]
+
+  Support for the following gems has been dropped. Metrics and events will no longer be created for these gems.
+    - Acts As Solr
```

Comment:
Instead of saying support for the following gems has been dropped.
-- Reuse the verbiage from the previous migration guide ----
The following gems had been previously deprecated and are now removed.

Reply:
[Complete](https://github.com/newrelic/docs-website/pull/11408/commits/49d803baca6c63737258b4919314ece1808fe8dd#r1109075863)